src/main/java/classify/commands/FileIOCommands.java

```diff
@@ -34,6 +34,7 @@ public static void writeStudentInfo(List <Student> list){
      */
     public static void readStudentInfo(ArrayList<Student> list) throws IOException{
         try{
+            System.out.println("Trying to load Student info");
```

Comment:
Good use of exception which will also help us if we encounter errors with file reading